bin/users.json

```diff
@@ -1,4 +1,9 @@
 [
   {"email": "master.caa@gmail.com", "roles": "caseworker,caseworker-caa", "lastName": "caa", "firstName": "master"},
-  {"email": "noc.approver@gmail.com", "roles": "caseworker-approver", "lastName": "noc", "firstName": "approver"}
+  {"email": "noc.approver@gmail.com", "roles": "caseworker-approver", "lastName": "noc", "firstName": "approver"},
```

Comment:
As well as `hmc-docker`, the `bootWithCCD` gradle plugin is also used in `hmc-cft-hearing-service`, so these users and roles would also need to be added to the Java class that configures users in that plugin

https://github.com/hmcts/hmc-cft-hearing-service/blob/master/src/cftlib/java/uk/gov/hmcts/reform/hmc/CftLibConfig.java